07_errors/hsy3418/error_test.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestInvalidInputError(t *testing.T) {
```

Comment:
You could refactor these into a table style test approach. All three of these have the same structure, where the only difference is the error code, error message, and output. Make these as variables inside your cases and you can repeat the logic.